Library/Formula/notmuch.rb

```diff
@@ -38,8 +40,17 @@ def install
     else
       args << "--without-emacs"
     end
-    system "./configure", *args
 
+    system "./configure", *args
     system "make", "V=1", "install"
+
+    if build.with? "python"
+      Language::Python.each_python(build) do |python, version|
+        Dir.chdir("bindings/python/") do
```

Comment:
If you're making that change anyway, I'd probably write this line as `cd "bindings/python" do` but not important